python/py-chm/Portfile

```diff
@@ -9,14 +9,18 @@ revision            1
 categories-append   textproc
 platforms           darwin
 maintainers         nomaintainer
+license             GPL
```

Comment:
We like to be more specific about which version(s) of the GPL is actually used. Some examples are GPL-2, GPL-2+, etc.

Reply:
Looking at the homepage, it should be `GPL-2+`.

Reply:
thanks Frank! You're right about the GPL-2+; I have changed it accordingly.